crates/sui-rpc-loadgen/src/main.rs

```diff
@@ -90,13 +91,13 @@ pub enum ClapCommand {
         #[clap(flatten)]
         common: CommonOptions,
     },
-    #[clap(name = "query-transactions")]
-    QueryTransactions {
-        #[clap(short, long)]
-        from_address: Option<String>,
+    #[clap(name = "query-transaction-blocks")]
+    QueryTransactionBlocks {
+        #[clap(long, parse(try_from_str), case_insensitive = true)]
+        address_type: AddressQueryType,
 
-        #[clap(short, long)]
-        to_address: Option<String>,
+        #[clap(long)]
```

Comment:
can you add a default value for this?

Reply:
actually, I think we should remove this and just reuse `data_directory`

Reply:
sounds good to me